### DISCUSSION TITLE
How to center an image based on a rectangle

### DISCUSSION THREAD

**Discussion by QC2168** [fabricjs/fabric.js#9425] [General]
I have two elements, one is rect and the other is image, and the image is inside the rect. I want to realize that the image is always centered based on the rect when the rect is moved

**Comment by QC2168** [comment]
I tried to use the group method, which could fulfill my requirements, but when I stretched the rect element, the image would also be stretched, and I wanted the image to remain the original size

**Comment by ShaMan123** [comment]
You are probably scaling the group - don't. Do something else or unscale the image on the  group's scaling event

**Reply by QC2168** [reply]
I don't know what you mean. How can I do that

**Reply by ShaMan123** [reply]
> unscale the image on the  group's scaling event

Look at examles and demos
